docs/releasenotes/release-1.126.md

```diff
@@ -22,10 +22,35 @@ TODO: list contributors with `git log v1.125.0... | grep Merge | grep from | awk
 
 ## Modified Beta Reconciliation
 
-We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
+We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The existing API is unchanged. To use the direct controller, add the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation to the corresponding CR object.
+
+* **`SecretManagerSecret`**
+
+  * #510 Enhanced `spec.rotation.nextRotationTime` to use a fixed datetime value to avoid relative `now()` friction. 
+  * #1081 Fixed the `spec.replication.auto` immutable issue
+  * #3051 Fixed the `spec.rotation.rotationPeriod` immutable issue 
+  * (WIP Pending#3282)Added the in-use version aliases in `status.observedState.versionAliases`
+  * Resolved update stalling issues. 
+
+* **`SecretManagerSecretVersion`**
+  
+  * Resolved update stalling caused by `DependencyNotReady` errors.
+  * Fixed the friction in `spec.enabled` that enabling/disabling a secret version does not take effect in GCP. 
+
+  * **API Behavior Change** 
```

Comment:
Should this be `##` ?   It feels very important, even if we're introducing it gradually, we still want feedback.

Reply:
Sure, I can make it a top tier item.  I put it under the `Modified Beta Reconciliation` because it only happens when user opt-in the direct reconciler.